### DISCUSSION TITLE
Is Standard dead?

### DISCUSSION THREAD

**Discussion by radek-duchon** [standard/vscode-standardjs#538] [General]
Hello. We have always used ESLint in our projects, but I have recently found `standard` and `ts-standard` and I liked how many problems it solved for us with very easy installation and (no) configuration. But I have also noticed that both projects and this extension as well have last commits year(s) ago and many opened issues. I belive I have found a bug in the extension but should I even bother reporting it?

**Comment by voxpelli** [comment]
See https://github.com/standard/standard/issues/1948 for some of the most up to date discussion on this

**Reply by radek-duchon** [reply]
Thanks @voxpelli.
